pivorakmeetup/pivorak-web-app
# Improve check in with qr code feature

### Description

- add qr code into the email body - ![email](https://dzwonsemrish7.cloudfront.net/items/1f0j230s0z0v3e3D2e0k/%5B19cca338561b44b37e6b6ce57328df59%5D_Image%2525202018-10-05%252520at%25252010.41.39%252520AM.png?v=b9874b59)
- increase the size of the qr code in the PDF - 
<img width="1280" alt="captura de pantalla 2018-10-05 a las 10 44 11 a m" src="https://user-images.githubusercontent.com/5591973/46522515-9de77080-c88b-11e8-9017-3ba4157efd44.png">
- replace markdown with slim template

### How to test instructions
- Visit http://localhost:3000/emails/en/visit_request_mailer_preview-attendance_confirmed
- Check out the email body
- Download the PDF
